statler_site/templates/play-detail-page.html

```diff
@@ -64,6 +66,11 @@
       <div id="details"></div>
       <div style="height: 200px; border-style: solid;">
         Review post box will go here.
+        <form action="{% url 'api:play-reviews' play_id %}" method="post">
+          {% csrf_token %}
+          <input type="text" name="text" />
+          <input type="submit" value="Post" />
+        </form>
```

Comment:
@aseebart I added a really basic post box so I could test my POST endpoint. Feel free to change it. The `{% csrf_token %}` is necessary and is used by django to prevent cross-site request forgeries.